engine/schema/src/main/java/com/cloud/storage/dao/VolumeDaoImpl.java

```diff
@@ -72,8 +79,14 @@ public class VolumeDaoImpl extends GenericDaoBase<VolumeVO, Long> implements Vol
     protected GenericSearchBuilder<VolumeVO, SumCount> primaryStorageSearch2;
     protected GenericSearchBuilder<VolumeVO, SumCount> secondaryStorageSearch;
     private final SearchBuilder<VolumeVO> poolAndPathSearch;
+    protected SearchBuilder<VolumeVO> volumePoolNotInClusterSearch;
+
     @Inject
     ResourceTagDao _tagsDao;
+    @Inject
+    HostDao _hostDao;
+    @Inject
+    VMInstanceDao _vmDao;
```

Comment:
I would encourage to not use the old pattern of using `_` for private class variables, as it is not part of the Java naming conventions. TBH, we should probably update the CloudStack [coding conventions](https://cwiki.apache.org/confluence/display/CLOUDSTACK/Coding+conventions) to reflect this as well.

Reply:
Done